### PR TITLE
Add RWT support

### PR DESCRIPTION
**Pull request by huaixv**

### CODE REVIEW

File: rwt.py [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+        exit()
+    return status
+
+
+if __name__ == '__main__':
+    r = learn_rwt("122.207.65.163", "USERNAME", "PASSWORD", "1")
+    r.LoginReading()
+    r.SectionID = "SECTION"
+    r.GetServerTime()
+    r.SetReadingResponseInformation('100', '1')
```
Comment (LiangYuxuan) [inline rwt.py]: this fixed test code should
* be removed, or
* accept cli input, or
* be commented out

I prefer the first one, but all of them are acceptable :)

Comment (huaixv) [reply rwt.py]: Test case removed

**Comment by LiangYuxuan** [comment]
Thank you!

**Comment by huaixv** [comment]
I submitted a sensitive data removal request to the GitHub support to remove the comment containing the network address. And this action requires the deletion of the entire pull request and your authorization.
I will send the email address associated with your account to the support,  as they have requested, if you approve this change.
Please let me know if your account email differs from the one in the commits or if you deny this request.

**Comment by LiangYuxuan** [comment]
Oh, that's fine, and I use the same email address on GitHub. :)

**Pull request merged**